test/integration/lib/foxboxHelper.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var helper = (function() {
+
+    function removeDB() {
+      var fs = require('fs');
+      var filePath = process.env.HOME + 
```

Comment:
Nit: [path.join()](https://nodejs.org/api/path.html#path_path_join_path1_path2) could be used. It normalizes the path to be cross-platform.